core/storage/blog/gae_models.py

```diff
@@ -334,16 +334,25 @@ def has_reference_to_user_id(cls, user_id):
             cls.editor_ids == user_id).get(keys_only=True) is not None
 
     @classmethod
-    def get_by_user(cls, user_id):
+    def get_by_user(cls, user_id, published=False, limit=None):
         """Retrieves the rights object for all blog posts assigned to given user
 
         Args:
             user_id: str. ID of user.
+            published: bool. True if rights model for published blog post are to
+                be fetched else false.
+            limit: int. Number of BlogPostRightsModel to be fetched. All
+                existing models will be fetched if none.
 
         Returns:
             list(BlogPostRightsModel). The list of BlogPostRightsModel objects
             in which the given user is a editor.
         """
+        if limit:
```

Comment:
if limit is not None

Reply:
Not required. If statement will not get executed if the value of arg is None.
Verified it again :) : 
![Screenshot from 2021-07-04 23-44-39](https://user-images.githubusercontent.com/68547101/124395904-63ed7680-dd24-11eb-9651-ecd7c89d4a60.png)

Reply:
"not null" and "true-y" are different things, actually. In this case the behaviour is different for limit=0. So, conceptually, I think you actually want "is not None".

Reply:
CHanged to 'is not None'